assets/downloads/jwt/jwt.js

```diff
@@ -0,0 +1,31 @@
+function generateJwtTest(env, clientToken, privateKey, keyId) {
+  var dt = new Date().getTime();
+  var uuid = 'xxxxxxxx-xxxx-4xxx-yxxx-xxxxxxxxxxxx'.replace(/[xy]/g, function (c) {
+    var r = (dt + Math.random() * 16) % 16 | 0;
+    dt = Math.floor(dt / 16);
+    return (c == 'x' ? r : (r & 0x3 | 0x8)).toString(16);
+  });
+
+  var data = {
+    "iss": clientToken.trim(),
+    "sub": clientToken.trim(),
+    "aud": env.trim() + "/v1/Token/auth",
```

Comment:
This hard-coded `v1` is going to be an issue

Reply:
@dhgreene i spoke to @Sun-Mountain about this yesterday. There is another version that she now has which handles the version. https://github.com/CMSgov/dpc-static-site/pull/58#discussion_r711304494

Reply:
The `jwt.hml` does indeed handle the version. That, paired with the ticket above, suffices for this PR. 👍